migrations/000048_AddRetries.up.sql

```diff
@@ -0,0 +1,25 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TYPE ImportFileStatus ADD VALUE 'RETRY';
+
+ALTER TABLE importfile
+    ADD COLUMN retries int DEFAULT 0;
+
+ALTER TABLE exportfile
+    ADD COLUMN retries int DEFAULT 0;
```

Comment:
why is this being added to exportfile it isn't being used?

Reply:
my mistake. fixed.